docs/installation.rst

```diff
@@ -20,6 +20,38 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
+
+|project_name| Repository Filesystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- No special filename requirements exist. Filenames may contain any valid
+  ascii character and be of any length. |project_name| stores data only 3
+  directory levels deep and uses short directory and filenames.
+- No special file metadata requirements exist regarding permissions or extended
+  file attributes. |project_name| only requires read/write permissions within
+  the repository filesystem.
+- Segment files generated by |project_name| are no larger than 5MB by default
+  (Ver. 1.0.x). This can be modified in :ref:`config-file`
+- There are no special requirements with regard to inode limits. Data is stored
+  within the segment files per the size specified in the config. Storing many
+  small files will not translate into having an equal or larger amount of files
+  within the repository. Deduplication helps as well. Most importantly, the
```

Comment:
Will review.
Deduplication helps because though you may try to store 10,000 files, you may not end up with that many segments as some files may share segments.  I can either clarify this statement or more likely remove it all together.

I was referencing this statement: "having a lot of small files in the backup source does not cause having the same or even bigger amount of files in the repo." I was trying to give an example of why that could be the case.

Also, I would like a little clarification if I'm mistaken, but it seems that if a file is smaller than a segment size, that files data may be combined with data from another file in the same segment, correct... or not?